packages/sit-onyx/src/components/OnyxTag/OnyxTag.ct.tsx

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from "../../playwright-axe";
+import OnyxTag from "./OnyxTag.vue";
+
+test("should render", async ({ mount, makeAxeBuilder }) => {
+  // ARRANGE
+  const component = await mount(<OnyxTag label="Tag" type="primary" />);
+
+  // ASSERT
+  await expect(component).toContainText("Tag");
+
+  // ACT
+  const accessibilityScanResults = await makeAxeBuilder().analyze();
+
+  // ASSERT
+  expect(accessibilityScanResults.violations).toEqual([]);
```

Comment:
Regarding your screenshot question:
With Playwright you can make screenshots like so (see [Playwright docs](https://playwright.dev/docs/screenshots)):

```ts
test("should render", async ({ mount }) => {
  // ARRANGE
  const component = await mount(<OnyxTag label="Tag" type="primary" />);

  // ASSERT
  await expect(component).toHaveScreenshot("default.png");
});
```

If you want to make a screenshot for every color, you could implement it like so:
```ts
import { ONYX_COLORS } from "../../types/colors";

ONYX_COLORS.forEach((color) => {
  test("should render", async ({ mount }) => {
    // ARRANGE
    const component = await mount(<OnyxTag label="Tag" type={color} />);

    // ASSERT
    await expect(component).toHaveScreenshot(`${color}.png`);
  });
});
```

Reply:
Note on the screenshots in CI:

- Screenshots will only be created/compared in the GitHub pipeline because they need to be generated/compared on a consistant operating system, e.g. comparing macOS with ubuntu screenshots will fail
- If you add screenshot tests, you need to run the [GitHub pipeline](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) manually for your branch so they are generated accordingly. If you want, you can just push them and I can trigger the pipeline for you.

Reply:
I pushed test cases with Playwright screenshot, could you have a look at them? 
(Please ignore the skipped tests, I did on purpose due to a violation of `WCAG 2 AA`: I put a comment in another thread)

Reply:
Looks good to me. Yes are are aware of the contrast issue and will work on that in #859

Reply:
That's good to know! I'll keep on tracking the issue! Thanks for the fast response @larsrickert !